### PR TITLE
Clean out some cruft in vg call

### PR DESCRIPTION
**Pull request by glennhickey**
* remove a bunch of outdated options and make defaults of remaining options a bit better
* report quality adjusted support in variant quality field
* makes bakeoff rocs look nicer in places, but no real significant gains, though nice to rely on fewer heuristics.  

One thing more I want to do is fix up the PASS/FAIL tag to be a bit smarter.  @adamnovak this is may also be a good place to implement the maximum coverage threshold you were talking about.

### CODE REVIEW

**Comment by adamnovak** [comment]
Yeah, I am thinking of renaming the current "FAIL" filter to something like "LOW_AD" and making a new depth-too-high filter.

How do these changes affect the `bakeoff.sh` F1 results? That's what I'm using to evaluate performance when I push stuff up.

I probably want to add my SV calling test to the Jenkins CI somehow, when I get a chance. For now I will have to see how these changes affect that manually.

**Comment by adamnovak** [comment]
Jenkins seems to be a bit upset:

```
INFO:toil.lib.docker:Calling docker with ['docker', 'run', '--rm', '--log-driver', 'none', '--env', 'TMPDIR=.', '-v', '/mnt/ephemeral/tmp/toil-aa7bbe87-9d61-463b-8109-df61f7669986/tmpM58O3m/68a52d38-4e6f-4ece-a0bb-5744e9727839/tIMoaRk:/data', '-w', '/data', '--entrypoint', '/bin/bash', 'jenkins-docker-vg-local', '-c', 'vg filter chunk_17_0.gam -x chunk_17_0.xg -t 1 -r 0.9 -fu -s 1000 -o 0 -q 15 | vg pileup chunk_17_0.vg - -t 8 -q 10 -a']
/bin/bash: line 1:     5 Broken pipe             vg filter chunk_17_0.gam -x chunk_17_0.xg -t 1 -r 0.9 -fu -s 1000 -o 0 -q 15
         6 Aborted                 (core dumped) | vg pileup chunk_17_0.vg - -t 8 -q 10 -a
```

It looks like maybe the filtering threw a segfault (or failed an assert).

File: test/t/18_vg_call.t [hunk 1]
```diff
@@ -6,21 +6,18 @@ BASH_TAP_ROOT=../deps/bash-tap
...
 vg view -J -l call/pileup.json -L > tiny.vgpu
-vg call tiny.vg tiny.vgpu -A calls_l.vg -s 10 -d 10 -q 10 -b 0.25  > /dev/null 2> /dev/null
+vg call tiny.vg tiny.vgpu -A calls_l.vg   > /dev/null 2> /dev/null
 is $? "0" "vg call doesn't crash"
 
 rm -f calls_l.json calls_l.vg
 
-vg view -J -l call/pileup.json -L | vg call tiny.vg - -A calls_l.vg -s 10 -d 10 -q 10 -b 0.25 -f 0.2 > /dev/null 2> /dev/null
-vg view -j calls_l.vg | jq . > calls_l.json
-is $(jq --argfile a calls_l.json --argfile b call/truth_l.json -n '($a == $b)') true "vg call -l produces the expected output for toy snp test case."
```
Comment (adamnovak) [inline test/t/18_vg_call.t]: Did you want to kill this test? Or did you want to replace the expected output?

**Comment by glennhickey** [comment]
That sounds good (splitting off FAIL filter).  I'd also like to play with
incorporating the binned baseline coverages for this (instead of hard
cutoff value).

Yes, Jenkins caught something!  For those bakeoff results, this PR
shouldn't have any effect (apart from breaking scripts due to options being
removed), as the calls and PASS/FAIL tag are untouched.

I'll fix up the Jenkins tests.  The whole CGL Jenkins server vanished the
other day, and I just recreated the vg project now.  I'll also add some
less trivial tests.

**Comment by glennhickey** [comment]
Kill. That test was (poorly) testing a bunch of logic in caller that's long
moved to call2vcf.  Think the Jenkins tests will do a far better job.

**Comment by glennhickey** [comment]
Will merge this now, and add more Jenkins tests (as promised above) in another PR.

**Pull request merged**